docs/README.md

```diff
@@ -55,6 +55,24 @@ All blocks exported by `store-link` share the same props:
 | `label`   | `string` | Link text                                                                                                                               | `undefined`   |
 | `href`    | `string` | Link URL                                                                                                                                | `'#'`         |
 | `target`  | `string` | This prop works the same way as the target of the [anchor element of html](https://developer.mozilla.org/pt-BR/docs/Web/HTML/Element/a) | `undefined`   |
+| `displayMode` | `DisplayMode` | How the link should be displayed | `'anchor'` |
+| `variant` | `Variant` | The variant to be used if `displayMode` is `'button'` | `'primary'` |
```

Comment:
Missing `size`. I think it would be better to have an object just for the button props

```js
buttonProps: {
  size: "small",
  variant: "primary"
}
```

What do you think?

Reply:
I think is a good idea, but about the size prop I didn't want to add to the documentation because I wanted to test it better and add in another PR, but thinking right now doesn't seem like a lot of work :thinking:

Reply:
Done